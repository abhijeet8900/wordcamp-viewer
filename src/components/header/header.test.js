import { render, screen } from "@testing-library/react";
import React from "react";
import Header from "./index.js";

describe("Header tests", () => {


  it("Renders Loation markers ", () => {
    render(<Header />);
      const marker = screen.getByText(/WordCamp Viewer/i);
      expect(marker).toBeInTheDocument();
     });

  it("Snapshot test ", () => {
    const { container } = render(<Header  />);
    expect(container).toMatchSnapshot();
  });
});
