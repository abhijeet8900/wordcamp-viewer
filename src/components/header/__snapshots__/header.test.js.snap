// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests Snapshot test  1`] = `
<div>
  <a
    class="no-underline flex"
    href="https://central.wordcamp.org/"
  >
    <h1
      class="w-full flex justify-center items-center text-primary font-bold text-center text-6xl my-4 leading-normal"
    >
      <span />
       
      WordCamp Viewer
    </h1>
  </a>
</div>
`;
